packages/react/src/Select/SelectPlain.tsx

```diff
@@ -0,0 +1,55 @@
+import { forwardRef } from 'react';
+import { ChevronDown } from '@obosbbl/grunnmuren-icons';
+import { cx } from '@/utils';
+
+export interface SelectPlainProps
+  extends Omit<React.ComponentPropsWithoutRef<'select'>, 'size'> {
+  /**
+   * Collection of <option />-elements
+   */
+  children: React.ReactNode;
+
+  /** Render select as invalid. Sets `aria-invalid` to true */
+  isInvalid?: boolean;
+
+  /**
+   * Changes font-size, padding and gaps
+   * @default medium
+   */
+  size?: 'medium' | 'small';
+}
+
+export const SelectPlain = forwardRef<HTMLSelectElement, SelectPlainProps>(
+  (props, ref) => {
+    const { children, className, isInvalid, size, ...rest } = props;
+
+    const isSmall = size === 'small';
+
+    return (
+      <div
+        className={cx(className, 'relative', isSmall ? 'text-sm' : undefined)}
```

Comment:
```suggestion
        className={cx(className, 'relative', isSmall ? { 'text-sm': isSmall }}
```

Personlig preferanse, men når det først bare er enten small eller undefined, så ser dette penere ut